install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -571,6 +571,9 @@ data:
 {{- if hasKey .Values.k8s "requireIPv6PodCIDR" }}
   k8s-require-ipv6-pod-cidr: {{ .Values.k8s.requireIPv6PodCIDR | quote }}
 {{- end }}
+{{- if and .Values.endpointStatus .Values.endpointStatus.enabled }}
```

Comment:
Wouldn't `{{- if .Values.endpointStatus.enabled }}` be enough here?

Reply:
Yes, it would. Updating.